go/launchd/launchd.go

```diff
@@ -168,6 +168,10 @@ func (s Service) Restart(wait time.Duration) error {
 
 // WaitForStatus waits for service status to be available
 func (s Service) WaitForStatus(wait time.Duration, delay time.Duration) (*ServiceStatus, error) {
+	if wait <= 0 {
```

Comment:
Fixing bug where nil status was being returned if wait = 0.
This wasn't causing any issues though, I just noticed it cause things to be reported wrong in the logs abount the install status for services.